src/WPF/WPF.Viewer/Samples/LocalServer/LocalServerGenerateElevationProfile/LocalServerGenerateElevationProfile.xaml.cs

```diff
@@ -593,5 +612,39 @@ private void CancelJobButton_Click(object sender, RoutedEventArgs e)
                 MyClearResultsButton.Visibility = Visibility.Visible;
             }
         }
+
+        private async void setTempPathButton_Click(object sender, RoutedEventArgs e)
+        {
+            try
+            {
+                // Local Server must be stopped to change the data path.
+                await LocalServer.Instance.StopAsync();
+
+                // Set the data path to content of the TextBox input by the user.
+                string userTempDataPath = tempPathBox.Text;
+                Directory.CreateDirectory(userTempDataPath); // CreateDirectory won't overwrite if it already exists.
```

Comment:
Our comment styling convention: "Comments should always be on their own line (don't place at the end of a line of code)."